packages/cmd/src/utils/CommandUtils.ts

```diff
@@ -74,28 +73,25 @@ export class CommandUtils {
         return command.trim();
     }
 
-
     /**
      * Check if an option was specified by the user.
      * @param optionName - the option to check for.
-     * @param {Arguments} commandArguments: The arguments specified by the user.
+     * @param {ICommandArguments["args"]} args: The arguments specified by the user.
      * @param {ICommandDefinition} commandDefinition - the definition for the command
      * @returns {boolean} true: the option was specified by the user.
      *                    false: the option was omitted/set to false
      */
-    public static optionWasSpecified(optionName: string, commandDefinition: ICommandDefinition,
-                                     commandArguments: Arguments): boolean {
+    public static optionWasSpecified(optionName: string, commandDefinition: ICommandDefinition, args: ICommandArguments["args"]): boolean {
         const optionDef = CommandUtils.getOptionDefinitionFromName(optionName, commandDefinition);
-
         if (isNullOrUndefined(optionDef)) {
             // if it's not an option, it's not specified
             return false;
         }
         if ((optionDef.type as string) === "boolean") {
-            return commandArguments[optionName] === true;
+            return args[optionName] !== undefined;
         }
         else {
-            return !isNullOrUndefined(commandArguments[optionName]);
+            return !isNullOrUndefined(args[optionName]);
```

Comment:
I see this function is deprecated. Maybe we should think about "reimplementing" it, or using what [NodeJS suggests](https://nodejs.org/api/util.html#util_util_isnullorundefined_object)
`prop === undefined || prop === null`
I'm worried that they could take it out without any warning. :worried:

Reply:
Yeah, we should be replacing these.

Reply:
It's good for now though. No worries there.